### ISSUE TITLE
Get Window Elements triggers NotImplementedError

### ISSUE THREAD

**Issue by vx-i** [robocorp/rpaframework#344]
**Environment**
- Windows 10 Pro
- Chrome 96

**Prerequisite**
```
conda.yaml
---
channels:
  - conda-forge

dependencies:
  - python=3.8.12
  - pip=21.3.1

  - pip:
    - rpaframework==12.2.0


tasks.robot
---
*** Settings ***
Library    RPA.Desktop.Windows    WITH NAME    WIN
Library    String

*** Keywords ***
Save File As
    WIN.Set Windows Backend    backend=win32
    ${winList}    WIN.Get Window List

    FOR    ${app}    IN    @{winList}
        ${match}    Get Regexp Matches    ${app}[title]    ^Save As$
        IF  ${match}
            Log To Console    \n---
            Log To Console    ${app}[handle]
            Log To Console    ${app}[title]
            ${appid}    WIN.Connect By Handle
            ...    handle=${app}[handle]
            ...    parse_elements=True
            ...    windowtitle=Save As
            ${appdetails}     WIN.Get App    1
            Log To Console    ${appdetails}
            Exit For Loop
        END
    END
    
    ${winElem}    WIN.Get Window Elements
```
**Steps to Reproduce**
- Open Chrome
- Go to any URL
- Right click on website and choose 'Save As'

**Expected Behavior**
Should get a list with all window elements from 'Save As' dialog

**Actual Behavior**
Get NotImplementedError.

-------------------
This issue even pops up earlier -> WIN.Connect By Handle. Except option parse_elements is set to False. If I understand the code correct, the root cause is here:

    def _parse_element_attributes(self, element: dict) -> dict:

In my case 'framework_id' is not valid and throw exception, but should be deleted anyway. See attributes_to_remove. Unfortunately it happens to late.

```
for attr in attributes_to_remove:
    element_dict.pop(attr, None)
```
should be replaced by something like
```
for attr in attributes_to_remove:
    try:
        element_attributes.remove(attr)
    except ValueError:
        pass
```
before
```
for attr in element_attributes:
```
After this change I get another exception here
```
        element_dict["legacy"] = (
            element.legacy_properties()
            if hasattr(element, "legacy_properties")
            else None
        )
```
I don't understand what this code do, therefore not sure to open a pull request. Maybe element.legacy_properties() should be encapsulated by try except block.

**Comment by vx-i** [comment]
Does anyone have any ideas on how I can move forward? Maybe pull request?

**Comment by mikahanninen** [comment]
@vx-i Sorry for the long delay. If you can make a pull request that would be great 👍

**Comment by vx-i** [comment]
#411

**Comment by cmin764** [comment]
Closing this as it seems to be released with **12.9.0**.
